amazon_cloud_code_generator/data/plugins/module_utils/core.py

```diff
@@ -408,12 +403,35 @@ def delete_resource(self, type_name: str, identifier: str) -> bool:
 
         return changed
 
+    def ensure_request_status(self, response: Dict) -> bool:
+        # Wait until reource request becomes IN_PROGRESS
+        response: Dict = {}
```

Comment:
This is just overwriting the `response` function argument. I assume that's not intended?

Reply:
Good point, thanks!